src/city/crud.py

```diff
@@ -0,0 +1,32 @@
+from sqlalchemy import select, insert
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from src.city.models import DBCity
+from src.city.schemas import CreateCity
+
+
+async def get_all_cities(db: AsyncSession):
+    query = select(DBCity)
+    result = await db.execute(query)
+    return [city[0]  for city in result.fetchall()]
```

Comment:
The list comprehension `[city[0] for city in result.fetchall()]` assumes that each row in the result contains only one element. This is correct if `DBCity` is the only column selected. Ensure this assumption holds true in all contexts where this function is used.